arch/c4_context.md

```diff
@@ -0,0 +1,49 @@
+# Level 1: Radix System Context
+
+This page implement the first "C" in the [C4Model](https://c4model.com/#coreDiagrams): the (System) **C**ontext.
+
+All production Systems (External and Radix) and Users that are commonly found on Radix networks are introduced on this page.
+
+![uncached image](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/radixdlt/docs/arch/arch/c4_context.puml)
+
+## Shards and Chunks
+
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/), shards enable Radix Networks to scale infinitely.
+This is achieved by splitting up global state (Ledger) to different nodes. Each node can be part of one or more Shards slices - called Chunks.
+
+The minimal Shard space that a nodes can serve is one Chunk. The number of Chunks served depends on the hardware configuration (CPU, MEM, I/O) of
+a specific node. Each node has advanced algorithms for monitoring its own performance and dynamically - to other nodes non-deterministically - scale up and down
+ the number of chunks served.
+
+### Redundancy
+
+Nodes within the same Shard space are redundant to each other, which means that they serve and validate the same transactions.
+
+## Core Nodes
+
+The first class citizens of the Radix Network. Nodes are organised in Shards. A node can serve ("be in") one or more Shard Chunks.
+
+Core Nodes are responsible for:
+
+* Validating all transactions that are relevant for its Chunks.
+* Store all transactions that are relevant for its Chunks.
+* Allow submission of atoms from Light Clients through a PubSubscribe-style API.
+* Dispatch ledger events to Light Clients that are subscribed.
+* Sync Ledger database with Peers in the same Shard Space.
+* Gossip about transactions with Peers in any/all Shards (structured gossip; not flooding).
+
+All nodes in the network are equal on start (same Mass). However, being a well-behaved citizen of a Radix Network, nodes gain mass 
```

Comment:
I believe we will have bootstrapped nodes with mass

Reply:
Practically that will be accomplished by us sending the first (200) transactions on each new network containing our own nodes.

Reply:
Based on prev convos with Dan, I think those first nodes will be bootstrapped with significantly more mass (via universe config).

Reply:
Yes, the bootstrapped mass will be in the Universe and applied to NIDs that we control.

I'm yet to calculate the value that we need, but likely those nodes will have significant power over the course of a year or so.

That power will dilute over time assuming that the network is marginally successful, the more successful it is the faster it will dilute.

Reply:
Thanks guys! I completely rewrote this page based on this new insight. @talekhinezh please read through it once more and confirm it make sense.